test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlEventSourceTests.netfx.cs

```diff
@@ -132,7 +132,7 @@ public async Task SuccessfulCommandTest(CommandType commandType, string commandT
 
             fakeSqlEventSource.WriteEndExecuteEvent(objectId, compositeState, sqlExceptionNumber);
 
-            Assert.Equal(2, activityProcessor.Invocations.Count);
+            Assert.InRange(activityProcessor.Invocations.Count, 2, 3);
```

Comment:
Windows and Linux are responding different values sometimes. Normally 2 (start/end) and sometimes, 3 (start/end/dispose).

Reply:
And, it is only happening for our fake tests

Reply:
Should we explicitly do shutdownSignal.Dispose() before validating? If we dont call it, the shutdownsignal.Dispose() is called afterwards, I suppose?

Reply:
thanks! that did the trick!